satpy/tests/reader_tests/test_abi_l1b.py

```diff
@@ -95,13 +117,12 @@ def _create_fake_rad_dataset(rad=None):
             "nominal_satellite_subpoint_lat": np.array(0.0),
             "nominal_satellite_subpoint_lon": np.array(-89.5),
             "nominal_satellite_height": np.array(35786.02),
-            "earth_sun_distance_anomaly_in_AU": np.array(0.99)
+            "earth_sun_distance_anomaly_in_AU": np.array(0.99),
```

Comment:
I disagree completely. I'm not sure how commas at the end of a line do or do not follow human readable language. A non-code list that is separated one per line would probably be bulleted anyway. I also disagree with code readability requiring closeness to verbal language. Code is not speech.

I've already given my reasons for why trailing commas help diffs and produce a similar looking lines between elements of the sequence. Having a trailing comma that almost maybe in the smallest way looks more like prose, does not remove the `diff` benefit of having it.

Reply:
I'm also not sure I can agree with any readability argument at the level of "it looks different/funny" because, as I pointed out in the black discussion, humans can get used to it. The diff argument has a real effect especially when git blame is involved.

Reply:
As explained on slack, I'm not going to complain about this anymore even if I don't agree with it.